lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -100,6 +100,11 @@ Function Choco-IsInstalled
         [string]$package
     )
 
+   if ($package -like "all")
```

Comment:
Why use `-like`, and not just:

if ($package -eq "all") {
    return $true
}